src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -36,12 +36,30 @@ const TopicPage = ({ pageData }) => {
     dateCreated: promo.firstPublished,
   }));
 
+  const getTranslations = () => ({
+    pageXOfY: 'Page {x} of {y}',
+    previousPage: 'Previous Page',
+    nextPage: 'Next Page',
+    page: 'Page',
+    ...translations.pagination,
+  });
+
+  const { pageXOfY, previousPage, nextPage, page } =
+    getTranslations(translations);
+
+  const translatedPage = pageXOfY
+    .replace('{x}', activePage)
+    .replace('{y}', pageCount);
+
+  const pageTitle = `${title}, ${translatedPage}`;
+
   return (
     <Wrapper role="main">
       <ATIAnalytics data={pageData} />
       <ChartbeatAnalytics data={pageData} />
       <MetadataContainer
-        title={title}
+        title={activePage >= 2 && pageCount > 1 ? pageTitle : title}
```

Comment:
`pageCount` is always >= `activePage`
```suggestion
        title={activePage >= 2 ? pageTitle : title}
```